### PR TITLE
feat(ui-ux): display preferred label and customize wallet list order

### PR DESCRIPTION
**Pull request by JoshuaC817**
#### Additional comments?:
as titled

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] No console errors on web
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: packages/ord-connect/src/components/SelectWalletModal/index.tsx [hunk 1]
```diff
@@ -24,12 +31,20 @@ import {
...
 import { waitForUnisatExtensionReady } from "../../utils/unisat";
 
-import { WalletButton } from "./WalletButton";
+import { WalletButton, WalletButtonProp } from "./WalletButton";
+
+interface WalletListItemProp extends WalletButtonProp {
+  isAvailable: boolean;
+}
+
+export type WalletOrderType = [Wallet, Wallet?, Wallet?, Wallet?, Wallet?];
```
[the remark this answers is not in the document]
Comment (thedoublejay) [reply packages/ord-connect/src/components/SelectWalletModal/index.tsx:40]: What is this? Haha

[the remark this answers is not in the document]
Comment (JoshuaC817) [reply packages/ord-connect/src/components/SelectWalletModal/index.tsx:40]: image

basically a type used for custom orders, wanting it to limit to 5 lol

[the remark this answers is not in the document]
Comment (JoshuaC817) [reply packages/ord-connect/src/components/SelectWalletModal/index.tsx:40]: unless we wanna make it simple then can just pass `Wallet[]`

[the remark this answers is not in the document]
Comment (thedoublejay) [reply packages/ord-connect/src/components/SelectWalletModal/index.tsx:40]: Ah yeah don't do that. If we add more wallets will be confusing haha

[the remark this answers is not in the document]
Comment (JoshuaC817) [reply packages/ord-connect/src/components/SelectWalletModal/index.tsx]: wokay removed

File: packages/ord-connect/src/components/SelectWalletModal/index.tsx [hunk 2]
```diff
@@ -396,6 +411,107 @@ export function SelectWalletModal({
...
+        renderAvatar,
+        isAvailable: !isMobile,
+      },
+    ];
+
+    if (!walletsOrder) {
+      return walletList;
+    }
+
+    const newList = walletsOrder.reduce<WalletListItemProp[]>(
```
Comment (thedoublejay) [inline packages/ord-connect/src/components/SelectWalletModal/index.tsx:474]: Wait I think this logic can be simpler.

1. We just pass the wallet (e.g Wallet.OKX, Wallet.Leather etc.)
2. Then we just do sort of walletList based on the walletOrder. Something like walletList.sort((a, b) => walletsOrder.includes(a)). Need to check the sorting logic I just typed it now haha

File: packages/ord-connect/src/components/SelectWalletModal/index.tsx [hunk 3]
```diff
@@ -396,6 +412,94 @@ export function SelectWalletModal({
...
+        order: 24,
+      },
+    ];
+
+    if (!walletsOrder) {
+      return walletList.sort((a, b) => a.order - b.order);
+    }
+
+    const updatedList = walletList.map((walletItem) => {
+      const foundIndex = walletsOrder.findIndex(
```
Comment (thedoublejay) [inline packages/ord-connect/src/components/SelectWalletModal/index.tsx:481]: Why do we need this part?

Comment (JoshuaC817) [reply packages/ord-connect/src/components/SelectWalletModal/index.tsx:481]: to get new index if the wallet was passed in through the array

Comment (thedoublejay) [reply packages/ord-connect/src/components/SelectWalletModal/index.tsx:481]: ahh then I don't think we need to sort it on the line above. Just return walletList directly

**Pull request merged**